.vscode/launch.json

```diff
@@ -1,17 +1,22 @@
 {
   "version": "0.2.0",
   "configurations": [
-    {
-      "type": "PowerShell",
-      "request": "launch",
-      "name": "ps: Interactive Session",
-      "cwd": "${workspaceRoot}"
+  {
+    "name": "MvcSandbox",
+    "type": "coreclr",
+    "preLaunchTask": "MvcSandbox-build",
+    "request": "launch",
+    "program": "${workspaceFolder}/artifacts/bin/MvcSandbox/Debug/net6.0/MvcSandbox.dll",
```

Comment:
Having the target framework here is problematic. We'd have to remember to update this every time we rev the TFM and it's easy to miss. Is there an alternative to this?

Reply:
Also, I wouldn't expect us to have any MVC specific tasks or actions outside of the MVC folder. We rarely expect developers to open up the solution in entirety.

Reply:
Unfortunately, the C# extension for VS Code requires that you provide a full-path to the assembly in order to debug it.

VS Code only picks up the launch config in the root `.vscode` directory of the workspace. So we can move this config to the analyzers project, but it would mean that users would have to open that specific folder to load the config. That's probably a safe assumption to make.

Reply:
@pranavkm I guess we can do that is there any env variable or path variable that we are using for `TargetFramework` in the aspnetcore repo, so I will check if VScode can pick up that so that we don't have to hardcode TFW everytime!

Reply:
@captainsafia I am finding a way (if there any) to utilize a single `.vscode` folder which is at the root. So although if anyone opens a subfolder of the repo, they should see debug options from that `vscode` folder. 
Right now, in this PR I have added `.vscode` folder in root as well and in the most commonly use `Mvc` folder.

Reply:
I am finding a way to read this value in tasks.json so we can utilize that there. 

https://github.com/dotnet/aspnetcore/blob/c925f99cddac0df90ed0bc4a07ecda6b054a0b02/eng/Versions.props#L45

Reply:
> I am finding a way to read this value in tasks.json so we can utilize that there.

If it's feasible to read from that property file (or maybe an environment variable), in the `launch.json` file that would be great!

> I am finding a way (if there any) to utilize a single .vscode folder which is at the root. So although if anyone opens a subfolder of the repo, they should see debug options from that vscode folder.

Got it. Hopefully, there's a strategy for this. I'm not aware of anything off the top of my head.

Thanks for taking the time to work on this!

Reply:
> Got it. Hopefully, there's a strategy for this. I'm not aware of anything off the top of my head.

Initial research on this found that it's not possible even today! Seems like we need to have a separate `.vscode` folder for samples OR for most likely projects which we all run on a regular basis for development for example `MvcSandbox`. 

> Thanks for taking the time to work on this!

Oh, please! My intension is to help all of you guys with simple bugs or problems as much as I can so that you can focus on bigger features! Hehe 😁😁

Reply:
@MCCshreyas I figured as much. For now, it's fine to place a .vscode directory where ever there is a `build.cmd` file.

Thanks for your help! Much appreciated!

Reply:
I will get that added and tested in this PR in upcoming commits!

Thanks! 🙌

Reply:
Considering that we can now read the target framework from an environment variable, let's update this to `${env:TARGET}`.